content/components/action-bar.mdx

```diff
@@ -223,10 +278,16 @@ An action bar has an ARIA role of [`toolbar`](https://developer.mozilla.org/en-U
 | `End`              | Moves focus to the **last** button.                                                                                                                                                                                                                                                                                                                    |
 | `Enter` or `Space` | Triggers the button **action**.                                                                                                                                                                                                                                                                                                                        |
 
+The `toolbar` role should be overridden if you're using an action bar in a context where the behavior of left arrow, right arrow, "Home", and "End" keys conflict with their behavior in a `toolbar`. For example, as list item actions within a list where focus may be moved between items using arrow, "Home", and "End" keys.
+
+In cases where we don't use the `toolbar` role, no role is applied and each buttons has one tab stop (their default keyboard navigation behavior).
+
 <!-- TODO: Add video -->
 
 ### Touch targets
 
+When the user is on a device with coarse pointer (such as a touch screen), the button target needs to be enlarged so users can easily tap the correct target button.
+
```

Comment:
How does the touchtarget work with different recommended `ActionBar` sizes?

Reply:
I described a possible solution in the description of this issue: https://github.com/github/primer/issues/3078

I want to get opinions from other designers before committing to that direction.